examples/online-archives/main.tf

```diff
@@ -11,6 +11,10 @@ resource "mongodbatlas_online_archive" "users_archive" {
     expire_after_days = 2
   }
 
+  data_expiration_rule {
+    expire_after_days = 90
+  }
```

Comment:
having a single attribute within an object is simply to maintain consistency with the definition in Atlas API ([createOnlineArchive](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Online-Archive/operation/createOnlineArchive)).